packages/teams-js/test/public/liveShare.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { LiveShareHost } from '../../src/internal/liveShareHost';
+import { liveShare } from '../../src/public/liveShare';
+
+describe('MicrosoftTeams-LiveShare', () => {
+  describe('initialize', () => {
+    it('should throw error due to package not being found', async () => {
```

Comment:
Can you please add positive test cases for the functions defined in liveShare.ts?

Reply:
That's really difficult to do because of the fact the code is trying to dynamically load a package dependency. We we need two separate version of the unit tests, one with the package dependency and one without. We'd need to brainstorm how that would work.

Reply:
Discussed further offline. We'll work together post merging this PR to see how best to set up the tests so we can validate the positive scenarios.